lib/sidekiq-scheduler/utils.rb

```diff
@@ -112,8 +112,10 @@ def self.sanitize_job_config(config)
     def self.new_rufus_scheduler(options = {})
       Rufus::Scheduler.new(options).tap do |scheduler|
         scheduler.define_singleton_method(:on_post_trigger) do |job, triggered_time|
-          SidekiqScheduler::Utils.update_job_last_time(job.tags[0], triggered_time)
-          SidekiqScheduler::Utils.update_job_next_time(job.tags[0], job.next_time)
+          if (job_name = job.tags[0])
```

Comment:
Not too sure why, but in some cases `job.tags == []`, which means we call `update_job_last_time(nil, triggered_time)`.

On redis-rb 4.x, `nil` would be silently casted to `""`, on 5.0 it raises (this is exactly the kind of errors I wanted to catch by making this change).

Reply:
thanks @casperisfine, I'll take a look to see why in some cases `tags` is an empty array (added to my list of things to do 😅)

Reply:
👌 if you want to repro, it happens twice in the test suite, but inside the Rufus thread, so somehow the Rufus thread crash, but the test pass.